ops/terraform/env/prod-sbx/migration/main.tf

```diff
@@ -15,7 +15,13 @@ module "migration" {
     tags              = {application="bfd", business="oeda", stack="prod-sbx", Environment="prod-sbx"}
   }  
 
-  bb      = 100
+  # These values control the CCS and HealthApt DNS record weight for each partner
+  # A value of:
+  #   100 - 100% CCS, 0% HealthApt
+  #    50 - 50% CCS, 50% HealthApt
+  #     0 - 0 CCS, 100% HealthApt
+  #
+  bb      = 50
```

Comment:
I thought we had BB on 100% traffic in prod-sbx?

Reply:
It's at 50% as a test of 50/50 load balancing.